src/components/pages/User/Registration.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {ReactElement, useEffect, useState} from "react";
+import {Alert, Col, Container, Row} from "react-bootstrap";
+import {notMinStrLength} from "../../../background/methods/checkInput";
+import fileFighter from "../../../assets/images/logos/adventurer-run.gif";
+import {registerNewUser} from "../../../background/api/registration";
+import {getWindowSize, getWindowSize_Interface} from "../../../background/methods/windowSize";
+import {getStyleValue} from "../../../background/methods/style";
+import {DEFAULT_ALERT_DURATION, MIN_PASSWORD_LENGTH} from "../../../background/constants";
+import UserInformationInput, {UserInformationInputInterface} from "./UserInformationInput";
+
+export default function Registration(): ReactElement {
+    const [alertMessage, setAlertMessage] = useState<string>("Error 404: No Message found.");
+    const [alertVariant, setAlertColor] = useState<"primary" | "secondary" | "success" | "danger" | "warning" | "info" | "light" | "dark">("success");
+    const [alertVisibility, setAlertVisibility] = useState<boolean>(false);
+
+    // const [newUser, setNewUser] = useState<UserInformationInterface>({
+    //     username: "",
+    //     password: "",
+    //     passwordConfirmation: ""
+    // })
+
+    const registrationContainer = document.getElementById("registrationContainer")
+    const logoSubmit = document.getElementById("logoSubmit")
+
+    useEffect(() => {
+        function repositionSubmitLogo() {
+            const logo = document.getElementById("logoSubmit")
+            if (logo) {
+                const container: HTMLElement | null = document.getElementById("registrationContainer");
+                const leftContainerOffset: number = container?.getBoundingClientRect().left ?? 0;
+
+                let containerPadding: string | number | null = getStyleValue(container, "padding-left");
+                const pxPosition = containerPadding.indexOf("px");
+                containerPadding = pxPosition === -1 ? null : Number(containerPadding.substr(0, pxPosition))
+
+                logo.style.left = -(leftContainerOffset + logo.offsetWidth * 2 + (containerPadding ?? 20)) + "px";
+            }
+        }
+
```

Comment:
wtf is this

Reply:
funny how this code was not even changed by this pr (only repositioned)

Reply:
but btw, it is the function to set the first position (depending on the screen width) of the moving image which is played when a new user is registered